src/main/java/org/gridsuite/merge/orchestrator/server/repositories/ProcessConfigEntity.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.merge.orchestrator.server.repositories;
+
+import lombok.Getter;
+import org.springframework.data.cassandra.core.mapping.PrimaryKey;
+import org.springframework.data.cassandra.core.mapping.Table;
+
+import java.util.List;
+
+/**
+ * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com
+ */
+@Getter
+@Table("configs")
+public class ProcessConfigEntity {
```

Comment:
This object is an entity AND also a DTO. I understand that it would be exactly the same object but for a cleaner separation of concern maybe we should have both.